### PR TITLE
rebuild site

### PR DESCRIPTION
**Pull request by t-kalinowski**
This is the result of just `devtools::document()` and `pkgdown::build_site()`, no changes to actual content.

### CODE REVIEW

File: website/pkgdown.yml [hunk 1]
```diff
@@ -1,65 +1,65 @@
...
-  reuters_mlp_relu_vs_selu: examples/reuters_mlp_relu_vs_selu.html
-  stateful_lstm: examples/stateful_lstm.html
-  text_explanation_lime: examples/text_explanation_lime.html
-  tfprob_vae: examples/tfprob_vae.html
-  unet: examples/unet.html
-  unet_linux: examples/unet_linux.html
-  variational_autoencoder: examples/variational_autoencoder.html
-  variational_autoencoder_deconv: examples/variational_autoencoder_deconv.html
-  vq_vae: examples/vq_vae.html
+  examples/addition_rnn: addition_rnn.html
```
Comment (dfalbel) [inline website/pkgdown.yml:8]: Maybe investigate why this is really needed? Is it a pkgdown change?

**Review by dfalbel — APPROVED** [review]
Does the file: website/reference/Rplot001.png is generated in process? Is seems to be an empty plot.

File: website/pkgdown.yml (hunk 1 above)
Comment (t-kalinowski) [reply website/pkgdown.yml:8]: I'm not sure it's needed. It's a change that `pkgdown::build_site()` made. Maybe I should to rollback to a previous version of pkgdown?

Comment (dfalbel) [reply website/pkgdown.yml:8]: I think that's fine them. All pages seems to still exist and the their path didn't change either.

**Pull request merged**